zerver/actions/realm_linkifiers.py

```diff
@@ -44,7 +45,7 @@ def do_remove_linkifier(
     if pattern is not None:
         RealmFilter.objects.get(realm=realm, pattern=pattern).delete()
     else:
-        RealmFilter.objects.get(realm=realm, id=id).delete()
+        RealmFilter.objects.get(realm=realm, id=assert_is_not_none(id)).delete()
```

Comment:
I think I'd prefer just `assert id is not None` on its own line at the start of the `else` clause.

Reply:
This probably applies to the other uses of `assert_is_not_none` in this PR.